recipes/dropstackframe/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "dropstackframe" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: dropstackframe
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dropstackframe-{{ version }}.tar.gz
+  sha256: 8f0b2724e729fee66d78ffaf15cfa1764e3a480f8c90f8dd6ed0e2b26c841c1a
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - poetry-core
+    - python
```

Based on [this line]( https://github.com/jesnie/dropstackframe/blob/e7f37dc46d4ee197f007ec7fe4b18654f3d7dd27/pyproject.toml#L24-L25 ) looks we need Python 3.7+

For `noarch: python`, we need this requirement in `host` and `run` like so

```suggestion
    - python >=3.7
```